homeassistant/components/nut/const.py

```diff
@@ -37,6 +37,14 @@
     "TEST": "Battery Testing",
 }
 
+AMBIENT_THRESHOLD_STATUS_OPTIONS = [
```

Comment:
This looks specific to the sensor platform so it should stay there

Reply:
Agreed, moved.